torch/csrc/jit/codegen/cuda/lower_replace_size.cpp

```diff
@@ -40,6 +40,10 @@ std::unordered_map<Val*, Val*> getSimplificationMap(Fusion* fusion) {
     if (id0->isBroadcast() || id1->isBroadcast()) {
       return;
     }
+    
+    if (id0->isGatherScatter() || id1->isGatherScatter()) {
```

Comment:
Is this the only place `IterType::GatherScatter` is used?

Reply:
Until now, yes. And this new iteration type handles the case where the dimension sizes of consumer and producer are not equal.